python/cuml/multiclass/multiclass.py

```diff
@@ -0,0 +1,225 @@
+        with cuml.internals.exit_internal_api():
+            preds = self.multiclass_estimator.predict(X)
+        return preds
+
+    @generate_docstring(return_values={'name': 'results',
+                                       'type': 'dense',
+                                       'description': 'Decision function \
+                                       values',
+                                       'shape': '(n_samples, 1)'})
+    def decision_function(self, X) -> CumlArray:
+        """ Calculate the decision function. """
+        X, _, _, _, _ = input_to_host_array(X)
+        with cuml.internals.exit_internal_api():
+            df = self.multiclass_estimator.decision_function(X)
+        return df
+
+    def get_param_names(self):
+        return super().get_param_names() + ['estimator', 'strategy']
+
+
+class OneVsRestClassifier(MulticlassClassifier):
+    """ Wrapper around Sckit-learn's class with the same name. The input can be
+    any kind of cuML compatible array, and the output type follows cuML's
+    output type configuration rules.
+
+    Berofe passing the data to scikit-learn, it is converted to host (numpy)
+    array. Under the hood the data is partitioned for binary classification,
+    and it is transformed back to the device by the cuML estimator. These
+    copies back and forth the device and the host have some overhead. For more
+    details see issue https://github.com/rapidsai/cuml/issues/2876.
+
+    For documentation see `scikit-learn's OneVsOneClassifier
+    <https://scikit-learn.org/stable/modules/generated/sklearn.multiclass.OneVsRestClassifier.html>`_. # noqa: E501
+
+    Parameters
+    ----------
+    estimator : cuML estimator
+    handle : cuml.Handle
+        Specifies the cuml.handle that holds internal CUDA state for
+        computations in this model. Most importantly, this specifies the CUDA
+        stream that will be used for the model's computations, so users can
+        run different models concurrently in different streams by creating
+        handles in several streams.
+        If it is None, a new one is created.
+    verbose : int or boolean, default=False
+        Sets logging level. It must be one of `cuml.common.logger.level_*`.
+        See :ref:`verbosity-levels` for more info.
+    output_type : {'input', 'cudf', 'cupy', 'numpy', 'numba'}, default=None
+        Variable to control output type of the results and attributes of
+        the estimator. If None, it'll inherit the output type set at the
+        module level, `cuml.global_output_type`.
+        See :ref:`output-data-type-configuration` for more info.
+    """
+    def __init__(self,
+                 estimator,
+                 *args,
+                 handle=None,
+                 verbose=False,
+                 output_type=None):
+        super(OneVsRestClassifier, self).__init__(
+            estimator, *args, handle=handle, verbose=verbose,
+            output_type=output_type, strategy='ovr')
```

Comment:
```suggestion
        super().__init__(
            estimator, *args, handle=handle, verbose=verbose,
            output_type=output_type, strategy='ovr')
```
You actually call `super` 2 different ways in this file. I think this is the better option (might be a personal opinion)